### ISSUE TITLE
Not catched error

### ISSUE THREAD

**Issue by hicTech** [modesty/pdf2json#200]
Parsing this pdf:
[test.pdf](https://github.com/modesty/pdf2json/files/4266654/test.pdf)
I notice "pdfParser_dataError" doesn't fire so, for this pdf, is impossible to handling error.
Any suggest?

**Comment by rprotazio** [comment]
I had same error, someone has a solution?

**Comment by xzhavilla** [comment]
This is the exception returned when opening the attached document via `pdf2json` cli:
```
Warning: Unhandled rejection: Error: bad XRef entry
Error: bad XRef entry
    at error (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:195:9)
    at XRef_fetch [as fetch] (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:5733:11)
    at XRef_fetchIfRef [as fetchIfRef] (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:5700:19)
    at Dict_get [as get] (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:4759:28)
    at Dict_getAll [as getAll] (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:4810:24)
    at PartialEvaluator_hasBlendModes [as hasBlendModes] (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:7006:29)
    at Object.eval [as onResolve] (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:4345:42)
    at Object.runHandlers (eval at <anonymous> (.../lib/pdf.js:64:1), <anonymous>:864:35)
    at listOnTimeout (internal/timers.js:554:17)
    at processTimers (internal/timers.js:497:7)
```

**Comment by zhongziqi** [comment]
any advices?

**Comment by modesty** [comment]
should work in `es6class` branch now

**Comment by modesty** [comment]
fixed since v2.0.0
